src/other/rjsf/InputWithInfoPopover.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import Typography from "@mui/material/Typography";
+import { Templates } from "@rjsf/mui";
+import { WidgetProps } from "@rjsf/utils";
+import React from "react";
+
+import InfoPopover, {
+    InfoPopoverProps,
+} from "../../mui/components/popover/info-popover/InfoPopover";
+import { PositionInfoPopover } from "./PositionInfoPopover.styled";
+
+const { BaseInputTemplate } = Templates;
+
+type InfoPopoverOptions = InfoPopoverProps & { content: string };
+
+export default function InputWithInfoPopover(props: WidgetProps) {
+    const { uiSchema } = props;
+    const infoPopover = (uiSchema ? uiSchema["ui:options"]?.infoPopover : {}) as InfoPopoverOptions;
```

Comment:
That's a very strange type casting here.
And later you use it with conditions like `infoPopover?.title` as it can be null or undefined

Reply:
Typed it more explicitly